### DISCUSSION TITLE
Adapter chez d’autres fournisseurs

### DISCUSSION THREAD

**Discussion by mrlulu51** [bokub/conso-api#59] [Idées]
Salut !

Je me demandais si il était possible d'adapter l'API pour accepter les autres applis de fournisseurs elec type comme EDF&Moi par exemple.

**Comment by bokub** [comment]
Salut, je pense que tu confonds un peu tout. 

Enedis n'est pas un fournisseur d'électricité mais un gestionnaire de réseau. Si tu as un compteur Linky chez toi, tu passes forcément par Enedis, quel que soit ton fournisseur d'électricité (y compris EDF)
